### PR TITLE
gym announcement draft

### PR DESCRIPTION
**Pull request by levkk**

### CODE REVIEW

File: pgml-docs/docs/gym/announcement.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+<br />
+
+<p align="center" markdown>
+  [Sign Up](https://gym.postgresml.org/signup/){ .md-button .md-button--primary .md-button }
+</p>
+
+
+## Notebooks for Postgres
+
+One big piece of novelty we think is exciting is our freshly designed Notebooks. They work much like the Jupyter Notebooks we love and use with Python, except ours are tailored towards building production-ready models with SQL.
```
Comment (montanalow) [inline pgml-docs/docs/gym/announcement.md:23]: I don't think this is a major selling point. Right now, our notebooks are arguably weaker than Jupyter.

File: pgml-docs/docs/gym/announcement.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+</div>
+</div>
+
+
+## Open Source
+
+PostgresML Notebooks, PostgresML itself, and our tutorials are open source and are available in our Github [repository](https://github.com/postgresml/postgresml). If you don't feel like joining our gym, nothing personal, you can run the entire experience from your laptop using our Docker deployment.
+
+
+## About PostresML
```
Comment (montanalow) [inline pgml-docs/docs/gym/announcement.md:68]: I think this needs to be part of the primary value prop explanation. I think no one will remember what PGML is exactly.

**Comment by montanalow** [comment]
Is this still in progress?

**Comment by levkk** [comment]
> Is this still in progress?

No not really. Maybe it should be

**Pull request closed without merging**